package.json

```diff
@@ -30,24 +30,24 @@
   },
   "peerDependencies": {
     "@nexus/schema": "^0.15.0",
-    "@prisma/client": "2.7.0",
+    "@prisma/client": "^2.8.1",
```

Comment:
Not sure if we prefer staying strict/conservative here. We technically support 2.8.0 & 2.8.1. I can also replace it by `2.8.0 || 2.8.1`

Reply:
Now that prisma's following semver (won't update to `2.9.x` if there's a breaking change), I think it's safe to specify `^2.8.x`

Reply:
We should match what we actually test, so straying strict right now.

Reply:
Updated to 2.8.0 || 2.8.1